lib/fastlane/plugin/firebase_app_distribution/actions/firebase_app_distribution_action.rb

```diff
@@ -209,16 +210,22 @@ def self.is_firebasecmd_supported?(cmd)
       end
 
       def self.get_upload_token(app_id, binary_path)
-        binary_hash = Digest::SHA256.hexdigest(File.open(binary_path).read)
+        begin
+          binary_hash = Digest::SHA256.hexdigest(File.open(binary_path).read)
+        rescue Errno::ENOENT
+          UI.crash!(ErrorMessage::APK_NOT_FOUND)
```

Comment:
Might be worth including the `binary_path` in the error message here somehow, that way the customer can see what we tried.